provision/core/wp-cli/provision.sh

```diff
@@ -17,20 +17,26 @@ function wp_cli_setup() {
     vvv_info " * Downloading wp-cli nightly, see <url>http://wp-cli.org</url>"
     curl -sO https://raw.githubusercontent.com/wp-cli/builds/gh-pages/phar/wp-cli-nightly.phar
     chmod +x wp-cli-nightly.phar
-    sudo mv wp-cli-nightly.phar /usr/local/bin/wp
+    mv wp-cli-nightly.phar /usr/local/bin/wp
 
     vvv_success " * WP CLI Nightly Installed"
 
     vvv_info " * Grabbing WP CLI bash completions"
     # Install bash completions
-    noroot curl -s https://raw.githubusercontent.com/wp-cli/wp-cli/master/utils/wp-completion.bash -o /srv/config/wp-cli/wp-completion.bash
+    mkdir -p /srv/config/wp-cli/
+    vvv_info " * Downloading WP CLI bash completions"
+    curl -s https://raw.githubusercontent.com/wp-cli/wp-cli/master/utils/wp-completion.bash -o /srv/config/wp-cli/wp-completion.bash
+    chown vagrant /srv/config/wp-cli/wp-completion.bash
   else
     vvv_info " * Updating wp-cli..."
     wp --allow-root cli update --nightly --yes
     vvv_success " * WP CLI Nightly updated"
   fi
-  vvv_info " * Installing WP CLI doctor sub-command"
-  wp --allow-root package install git@github.com:wp-cli/doctor-command.git
+  if [ "${VVV_DOCKER}" != 1 ]; then
```

Comment:
Why execute this only on non docker? Is this incompatible or useless in docker?

Reply:
Docker image builds vs Docker container scripts. We should not consider `VVV_DOCKER` to be final, think of it as an experimental internal temporary thing that's necessary until a proper system is in place